book/setting_up.md

```diff
@@ -65,24 +65,24 @@ how to build our GUI.
 # gui/CMakeLists.txt
 
 set(CMAKE_CXX_STANDARD 14)
-set(CMAKE_INCLUDE_CURRENT_DIR ON)
 
 set(CMAKE_AUTOMOC ON)
 set(CMAKE_AUTOUIC ON)
 set(CMAKE_AUTORCC ON)
+set(CMAKE_INCLUDE_CURRENT_DIR ON)
 find_package(Qt5Widgets)
 
-add_executable(gui 
-    main_window.cpp main_window.hpp main.cpp)
+set(SOURCE main.cpp)
```

Comment:
Don't we still need to include `main_window.cpp`?

Reply:
At this point of walking through the tutorial one had only created main.cpp (`touch main.cpp`) and added content to it, but not yet the `main_window.*` files. So when following the tutorial closely, when invoking cmake with the any of those files listed for SOURCE, cmake will fail.

One option might be to tell the reader to also already create/touch the other two files together with main.cpp, but it felt to me you want to introduce the details of mainwindow only at a later point for a reason, so creating those files before might be spoiling that. That's why I proposed the changes after "The first thing is to pull our main window out into its own source file[s]."

Reply:
> That's why I proposed the changes after "The first thing is to pull our main window out into its own source file[s]."

Yeah that sounds wise. It would feel a bit odd being told to create a couple empty files and add them into your `CMakeLists.txt` file when we aren't going to use them for a while.